addons/io/org.openhab.io.imperihome/README.md

```diff
@@ -0,0 +1,355 @@
+# ImperiHome integration service
+
+This IO service exposes OpenHAB Items to the Evertygo [ImperiHome](http://www.evertygo.com/imperihome) dashboard app for Android and iOS.
+It creates a REST service at ```/imperihome/iss``` that implements the [ImperiHome Standard System API](http://dev.evertygo.com/api/iss) (ISS).
+
+## Installation
+
+At the moment the ImperiHome integration service is not part of the OpenHAB distribution and can't be installed using the Paper UI.
```

Comment:
Please update this paragraph now that we are close to including it in the distro.